examples/search_examples/public/search/app.tsx

```diff
@@ -82,6 +82,9 @@ function formatFieldsToComboBox(fields?: DataViewField[]) {
   });
 }
 
+const bucketAggType = 'terms';
+const metricAggType = 'avg';
```

Comment:
It would be useful to have the ability to switch this to `median` in the search example UI. Maybe something for later?